### PR TITLE
Insist on Black formatting for Python code

### PR DESCRIPTION
**Pull request by mythmon**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -65,14 +65,16 @@ artifact: build pack docs
 	cp -r docs/out artifacts/docs
 
 lint: $(NPM_INSTALL_STAMP) build
-	$(ESLINT) .
-	$(TSC) --noEmit --project tsconfig.json
-	$(PRETTIER) --check .
+	# $(ESLINT) .
+	# $(TSC) --noEmit --project tsconfig.json
+	# $(PRETTIER) --check .
```
Comment (pdehaan) [inline Makefile:70]: LGTM, but we should make sure we uncomment out these other lint tasks before final merge.

**Comment by pdehaan** [comment]
Confirmed CI output matches the expected number of Python files in this project:

```sh
git ls-files '*.py'
python/mozilla_nimbus_shared/__init__.py
python/mozilla_nimbus_shared/test.py
```

```sh
# ./node_modules/.bin/eslint .
# ./node_modules/.bin/tsc --noEmit --project tsconfig.json
# ./node_modules/.bin/prettier --check .
cd python; poetry run black --check mozilla_nimbus_shared/test.py mozilla_nimbus_shared/__init__.py
All done! ✨ 🍰 ✨
2 files would be left unchanged.
```

File: Makefile (hunk 1 above)
Comment (mythmon) [reply Makefile:70]: Oops. Thanks for catching this, I'll fix it before merging.

**Comment by mythmon** [comment]
@jaredlockhart does that way work for you?

**Pull request merged**